sdk/attestation/azure-security-attestation/pom.xml

```diff
@@ -84,6 +84,12 @@
       <version>1.3.4</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
```

Comment:
Curious why you use bouncycastle here and not nimbus?

Reply:
I couldn't find working APIs in Nimbus to create X.509 certificates.